editor/src/components/canvas/ui-jsx-canvas.spec.tsx

```diff
@@ -2041,6 +2041,34 @@ export var ${BakedInStoryboardVariableName} = (props) => {
     </Storyboard>
   )
 }
+`,
+    )
+  })
+
+  it('React.useEffect at the root fails usefully', () => {
+    testCanvasRender(
```

Comment:
there's also a `testCanvasRenderInline` now that helps with readability (at the cost of making the file larger)